src/MCPClient/osdeps/RedHat-7.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
I expected there could be some small differences between RH and CentOS but so far this wasn't the case. We could probably use a symlink to one for now (and split into separate files if we find any difference in the future)